src/replace-libraries-ngpackagr-builder.ts

```diff
@@ -0,0 +1,63 @@
+import {versions} from './angular-versions';
```

Comment:
suggestion: Replace all occurrences of *Ngpackager* in this file with *Ngpackagr* to correct the spelling.